src/input.c

```diff
@@ -228,6 +255,31 @@ static void ascii_key(input_t *restrict input, char ch)
         doom_key = DOOM_KEY_ENTER;
     if (doom_key == '\n')
         doom_key = DOOM_KEY_ENTER; /*  Kitty sends LF for Enter */
+    if (doom_key == 'u') {
+        /* kitty_keyboard_protocol_active */
+        if (input->csi_u_params.keycode == 99 &&
+            input->csi_u_params.modifiers == 5) {
+            /* ctrl+c */
+            atomic_store_explicit(&input->exit_requested, true,
+                                  memory_order_relaxed);
+            return;
+
+        } else if (input->csi_u_params.keycode == 57442 ||
```

Comment:
What is 57442?

According to KKP spec, it's KP_Begin (numpad 5 without numlock). Add a comment or define:
```c
#define KKP_KP_BEGIN 57442  /* Numpad 5 without numlock */
```